usecases/scheduler/src/lambda/ssv2_configUpdater/configUpdater.py

```diff
@@ -0,0 +1,274 @@
+        "JobName": "ssv2-" + ssv2configId, 
+        "JobQueue": jobQueue, 
+        "ContainerOverrides": 
+            { "Environment": 
+                [ 
+                    { "Name": "PARAMS", "Value": ssparams }, 
+                    { "Name": "S3_OUTPUT_BUCKET", "Value": s3Bucket },
+                    { "Name": "CONFIG_ID", "Value": ssv2configId },
+                    { "Name": "CROSSACCOUNTS", "Value": crossAccounts}
+                ] 
+            } 
+    }
+
+    target = {
+        'Arn': 'arn:aws:scheduler:::aws-sdk:batch:submitJob',
+        'RoleArn': ebrolesARN,
+        'Input': json.dumps(inputJson)
+    }
+
+    flexibleTimeWindow = {
+        "Mode": 'OFF'
+    }
+
+    args = {
+        "Description": '[Created by SSv2] Screener Scheduler:' + ssv2configId,
+        "FlexibleTimeWindow": flexibleTimeWindow,
+        "Name": 'ScreenerScheduler-' + ssv2configId,
+        "ScheduleExpression": cronPattern,
+        "ScheduleExpressionTimezone": 'UTC',
+        "StartDate": datetime.now(),
+        "GroupName": schedulerGroupName,
+        "Target": target
+    }
+
+    print("Attempting to update EventBridge Scheduler: ...")
+    print(json.dumps(args, indent=4, default=str))
+    # scheduler.create_schedule(**args)
+
+    try:
+        scheduler.create_schedule(**args)
+    except botocore.exceptions.ClientError as e:
+        print(e)
+        if(e.response['Error']['Code'] == 'ConflictException'):
+            print("Calling scheduler.update_schedule...")
+            scheduler.update_schedule(**args)
+
+## update SNS reciepient
+def updateSnsRecipient(ssv2configId, emails):
+    existingSubscriptions = []
+    snsName = snsArnPrefix + '-' + ssv2configId
+
+    #create if not exists
+    resp = sns.create_topic(Name=snsName)
+    snsArn = resp.get('TopicArn')
+
+    args = {'TopicArn': snsArn}
+
+    while(True):
+        resp = sns.list_subscriptions_by_topic(**args)
+        existingSubscriptions.extend(resp.get('Subscriptions'))
+
+        NextToken = resp.get('NextToken')
+        if NextToken:
+            args['NextToken'] = NextToken
+        else:
+            break
+
+    for existingEmail in existingSubscriptions:
+        if existingEmail.get('Protocol') != 'email':
+            continue
+
+        if existingEmail.get('Endpoint') not in emails:
+            if existingEmail.get('SubscriptionArn') == 'PendingConfirmation':
+                print('Unable to remove {}, current status is PendingConfirmation. Amazon will delete automatically the subscription after 3 days of creation'.format(existingEmail.get('Endpoint')))
+                continue
+
+            print('Removing {}'.format(existingEmail.get('Endpoint')))
+            sns.unsubscribe(SubscriptionArn=existingEmail.get('SubscriptionArn'))
+
+        if existingEmail.get('Endpoint') in emails:
+            emails.remove(existingEmail.get('Endpoint'))
+
+    for email in emails:
+        print('Adding {}'.format(email))
+        sns.subscribe(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The Amazon SNS subscribe operation by default returns either the subscription ARN (if the subscribed endpoint is managed by AWS and it belongs to the same account as the topic) or the phrase: `PENDING CONFIRMATION`. If you want to always return the subscription ARN, set the `ReturnSubscriptionArn` argument to `True`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sns.html#SNS.Client.subscribe)